homeassistant/components/bbb_gpio/binary_sensor.py

```diff
@@ -28,9 +30,18 @@
     {vol.Required(CONF_PINS, default={}): vol.Schema({cv.string: PIN_SCHEMA})}
 )
 
+_LOGGER = logging.getLogger(__name__)
+
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Beaglebone Black GPIO devices."""
+    _LOGGER.warning(
```

Comment:
Should we move the warning to `setup` in the init module so there's only max one warning for the integration?

Reply:
Done